sonar-batch/src/main/java/org/sonar/batch/scan/ProjectScanContainer.java

```diff
@@ -108,7 +109,8 @@ public ProjectScanContainer(ComponentContainer globalContainer, AnalysisProperti
   @Override
   protected void doBeforeStart() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Protected method 'doBeforeStart(...)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUnusedProtectedMethod)